src/index.ts

```diff
@@ -215,36 +207,42 @@ async function notifyAutoCache() {
 
   try {
     core.debug(`Indicating workflow start`);
-    const res: any = await gotClient.post(`http://${core.getInput('listen')}/api/workflow-start`).json();
-    core.debug(`back from post`);
-    core.debug(res);
+    const res: Response = await gotClient
+      .post(`http://${core.getInput("listen")}/api/workflow-start`)
+      .json();
+    core.debug(`back from post: ${res}`);
```

Comment:
question: this lets you run MNC and it ... doesn't do anything until you call workflow-start?

Reply:
My memory is the workflow-start endpoint is intended to let MNC survey the Nix store to identify what paths are already there, so in the shutdown phase it can diff and upload everything new...

Reply:
aha, slick!